test/unit/lodash.spec.js

```diff
@@ -0,0 +1,605 @@
+                var obj = params[0];
+                var temp = obj;
+                var pattern = params[1];
+                _.unset(obj, pattern);
+                expect(obj).to.eql(temp);
+            });
+        });
+    });
+
+    describe('update', function() {
+        beforeEach(function() {
+            params = datasets.update();
+        });
+        it('should be an object', function() {
+            var element = params[0] instanceof Object;
+            var element1 = typeof params[1];
+            var element2 = params[2] instanceof Function;
+            expect(element).to.eql(true);
+            expect(element1).to.eql('string');
+            expect(element2).to.eql(true);
+        });
+        it('should double height', function() {
+            expect(_.update.apply(_, params)).to.eql({
+                'rectangle': [{
+                    'rect1': {
+                        'width': 4,
+                        'height': 8
+                    }
+                }]
+            });
+        });
+    });
+
+    describe('updateWith', function() {
+        beforeEach(function() {
+            params = datasets.updateWith();
+        });
+        it('should be an object', function() {
+            var element = params[0] instanceof Object;
+            var element1 = typeof params[1];
+            var element2 = params[2] instanceof Function;
+            expect(element).to.eql(true);
+            expect(element1).to.eql('string');
+            expect(element2).to.eql(true);
+        });
+        it('should triple height', function() {
+            expect(_.updateWith.apply(_, params)).to.eql({
+                'rectangle': [{
+                    'rect1': {
+                        'width': 4,
+                        'height': 12
+                    }
+                }]
+            });
+        });
+    });
+
+    describe('values', function() {
+        describe('values1', function() {
+            beforeEach(function() {
+                params = datasets.values1();
+            });
+            it('should be an object', function() {
+                var element1 = params[0] instanceof Object;
+                expect(element1).to.eql(true);
+            });
+            it('should have 1 parameter', function() {
+                expect(params.length).to.eql(1);
+            });
+            it('should not contain own property with value 2', function() {
+                var obj = params[0];
+                for (var key in obj) {
+                    if (obj.hasOwnProperty(key)) {
+                        expect(obj[key]).to.not.eql(2);
+                    }
+                }
+            });
+            it('has 2 own properties', function() {
+                var obj = params[0];
+                if (Object.keys(obj).length < 2 || Object.keys(obj).length > 2) {
+                    expect(Object.keys(obj).length).to.eql(2);
+                }
+            });
+            it('should return [null]', function() {
```

Comment:
From API:

Returns
(Array): Returns the array of property values.

????

I may understand what do you meant looking at the actual contents of the test but this title is completely misleading.